src/Sarif.Viewer.VisualStudio/FileAndForgetEventName.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Sarif.Viewer
+{
+    internal static class FileAndForgetEventName
+    {
+        private const string Prefix = "Microsoft/VisualStudioSarifViewer/";
+
+        private static readonly Lazy<string> s_infoBarCloseFailure = new Lazy<string>(() => MakeEventName("InfoBarClose/Failure"));
```

Comment:
>s_infoBarCloseFailure [](start = 45, length = 21)

same note, are we really buying anything with lazy initialization of two string constants?

Reply:
Now fixed in both projects.

---
In reply to: [520862775](https://github.com/microsoft/sarif-visualstudio-extension/pull/265#discussion_r520862775) [](ancestors = 520862775)